draft-ietf-acme-acme.md

```diff
@@ -1639,10 +1645,9 @@ that have been provisioned to a web server.
 The server is said to "finalize" the authorization when it has completed
 one of the validations, by assigning the authorization a status of "valid"
 or "invalid", corresponding to whether it considers the account authorized
-for the identifier.  If the final state is "valid", the server MUST add an
-"expires" field to the authorization.  When finalizing an authorization,
-the server MAY remove challenges other than the one that was completed. The
-server SHOULD NOT remove challenges with status "invalid".
+for the identifier. When finalizing an authorization, the server MAY remove
+challenges other than the one that was completed, and may modify the "expires"
```

Comment:
The intent here is to say that an authorization in the "valid" state MUST have an "expires" field.  So we should not remove this entirely.  How about:

```
If the final state is "valid", then the server MUST include an "expires" field in the authorization.
```

That doesn't presume that one was not already present.